src/design-system/asset-input/simple-asset-input.component.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { Box } from '../box';
+import { Text } from '../text';
+
+import { AmountInput } from './amount-input.component';
+import { AmountInputAlignments } from './amount-input.css';
+import * as cx from './simple-asset-input.css';
+
+import type { Asset, AssetState } from './asset-input.data';
+
+interface Props {
+  state: AssetState;
+  translations: {
+    balance: string;
+  };
+  onAmountChange?: (asset: Readonly<Asset>, amount: string) => void;
+}
+
+export const SimpleAssetInput = ({
+  state,
+  translations,
+  onAmountChange,
+}: Readonly<Props>): JSX.Element => (
+  <div className={cx.root}>
+    <Box className={cx.amountBox}>
+      <AmountInput
+        id={state.asset.id}
+        value={state.asset.amount}
+        alignment={AmountInputAlignments.left}
+        onChange={(value): void => {
+          onAmountChange?.(state.asset, value);
+        }}
+      />
+    </Box>
+    <Box className={cx.balance}>
+      <Text.Body.Normal color="secondary">
+        {translations.balance}: {state.asset.balance}
```

Comment:
Show we render a ticker? It is visible on the midnight send flow designs
<img width="300" alt="image" src="https://github.com/user-attachments/assets/da764e34-5f70-4d23-96d4-13d044f78d90">

Reply:
Good catch @szymonmaslowski